wolox-books/src/app/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpParams } from "@angular/common/http";
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  newUser: any;
+  ROOT_URL: string;
+
+  constructor(private http: HttpClient) {
+    this.ROOT_URL = 'https://wbooks-api-stage.herokuapp.com/api/v1/users';
+  }
+
+  createUser(user) {
+    this.newUser = this.http.post(this.ROOT_URL, user, {observe: 'response'})
```

Comment:
you don't need to asign the response to a variable, just subscribe to the response like you did.